netops_demo/py/functions/generate/generate.py

```diff
@@ -27,7 +28,6 @@ def generate(context, event):
 
 
 def init_context(context):
-
```

Comment:
Empty line before comment is a good thing

Reply:
Done